tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -424,6 +475,68 @@ def test_limit_and_offset(self, test_client, params, expected_asset_uris):
         asset_uris = [asset["uri"] for asset in response.json()["asset_events"]]
         assert asset_uris == expected_asset_uris
 
+    @pytest.mark.usefixtures("time_freezer")
+    @pytest.mark.enable_redact
+    def test_should_mask_sensitive_extra_logs(self, test_client, session):
```

Comment:
Not sure why we have `logs` in the name of the test, I don't think we assert any `Log` here.

Reply:
done